borg/platform_windows.pyx

```diff
@@ -190,6 +225,7 @@ def set_owner(path, owner, sidstring = None):
         domainlength = 0
         LookupAccountNameW(NULL, temp, newsid, &length, NULL, &domainlength, &sid_type)
         if GetLastError() != 0:
+            raise_error('LookupAccountNameW', owner)
             PyMem_Free(temp)
             return
```

Comment:
it doesn't look like it is ever coming back from raise_error. so, first free the memory?